app/src/interfaces/_system/system-language/system-language.vue

```diff
@@ -23,16 +23,24 @@ export default defineComponent({
 			type: String,
 			default: null,
 		},
+		includeProjectDefault: {
+			type: Boolean,
+			default: true,
+		},
```

Comment:
Why would we want to exclude the default language from the list of options? 🤔

Reply:
This is because of the settings page: `system-language` is used as interface for the Default/Project Language field. It does not make sense to be able to choose the "Default Language" option in the dropdown as value for the Default Language field.

Reply:
Ohh gotcha, yeah that makes sense 👍🏻 I thought the flag would control whether or not the _language_ would show up (so like it would exclude en-US from the list f.e.). Didn't realize there was a dedicated "Project Default" option in that interface 👍🏻